README.md

```diff
@@ -7,32 +7,52 @@
 
 > Share your conferences activity to your buddies
 
-## Installation
+## Requirements
 
-### Local
+This project requires:
 
-> Add `local.starfleet.app` to your hosts file
+- Docker 
+- [mkcert](https://github.com/FiloSottile/mkcert)
+- [pipenv](https://github.com/pypa/pipenv)
 
-Use [mkcert](https://github.com/FiloSottile/mkcert) to install a CA on your system to be able to generate a valid self-signed certificate for local.starfleet.app
+## Installation
 
-`$ mkcert *.starfleet.app`
+### Local
 
-> certificate and key must be placed in infrastructure/docker/services/router/etc/ssl/certs
+First, you will need to add `local.starfleet.app` to your hosts file.
+> `127.0.0.1 local.starfleet.app`
 
-Install Python virtualenv to be able to use fabric commands to control easily the Docker stack
+Install Python [pipenv](https://github.com/pypa/pipenv) to be able to use fabric commands to easily control the Docker stack.
```

Comment:
I re-read the whole README, and IMHO, we don't need to talk about python pipenv here, since we have already talked about it in the `Requirements` section.

TL;DR: You can simply remove this sentence.